TreeMaker/src/TreeMaker.cc

```diff
@@ -48,10 +48,32 @@ TreeMaker::TreeMaker(const edm::ParameterSet& iConfig) :
 	treeName = iConfig.getParameter<string>("TreeName");
 	doLorentz = iConfig.getParameter<bool>("doLorentz");
 	sortBranches = iConfig.getParameter<bool>("sortBranches");
-	//loop over all var type names to initialize TreeObjects
-	set<string> nameSet;
+
+	// parse the TitleMap
 	stringstream skipMessage;
 	stringstream message;
+	set<string> titleSet;
+	vector<string> titleVector = iConfig.getParameter< vector<string> >("TitleMap");
+	message << "TitleMap:\n";
+	for(unsigned i = 0; i < titleVector.size(); i+=2){
+		string variable = titleVector.at(i);
+		string title = titleVector.at(i+1);
+		message << "full name:" << variable << " -> title: " << title << "\n";
```

Comment:
since we already print the trigger list once in the logs, we probably don't need to print it 3 more times... maybe we can add another TreeMaker option like debugTitles and only print this if it is enabled?

Reply:
Done